### PR TITLE
feature: creating an account with a hashed password is complete

### LINKED ISSUE #6: ⚡ Account creation
Create account with the following user properties:

- E-mail: should end with '@hva.nl'
- Username: always unique
- Name: first, insertion, last
- Password
- Profile picture: users should be able to upload jpg or png files

When creating an account (aka inserting a new document into a collection), maybe you can reuse the CRUD operations I have written? For example:

```javascript
CRUD.createDocs(schemas.User, { required object properties })
.then((userObject) => {
       CRUD.createDoc(schemas.Student, { user: userObject.id, cmd_skills: { best: ["622a0cbb6df854b8951e7e55"], want_to_learn: ["622a0cbb6df854b8951e7e55"]}, classes: userObject.classes, courses: userObject.courses })
        .then((student) => {
            CRUD.addIdReferenceToDoc(schemas.Course, student.courses, "students", student.id);
            CRUD.addIdReferenceToDoc(schemas.Class, student.classes, "students", student.id);
        });
});
```

Lmk if you need help!

### PR DESCRIPTION
**Pull request by Aidan98**
Styling still missing though. Does anyone have suggestions for the design of the login and register page?

### CODE REVIEW

File: views/login.ejs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<body>
+    <%- include ('./partials/head') -%>
```
Comment (noyamirai) [inline views/login.ejs]: The `<head>` tag, in this case the partial, should be above the `<body>` tag !

File: views/partials/head.ejs [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-	<title>Team Creator</title>
+<!doctype html>
+<html lang="nl">
+	
+<head>
+	<meta charset="UTF-8">
+	<meta name="author" content="CMD Online">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+		
+	<link href="../../public/styles/style.css" rel="stylesheet">
```
Comment (noyamirai) [inline views/partials/head.ejs:9]: As discussed last night, this is causing issues for some reason, but like you said I think that once I convert my team generator stuff into routes it should (hopefully) solve itself hehe

File: views/register.ejs [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<body class="register_page">
+    <%- include ('./partials/head') -%>
```
Comment (noyamirai) [inline views/register.ejs]: As mentioned earlier, `<head>` should be above body tag

**Review by noyamirai — CHANGES_REQUESTED** [review]
Looks good! So to get the login/create account screens I would simply navigate to `/` right? 🤔  Also, I added some feedback on a few view pages regarding `<head>` tag placements, if you could change that I would appreciate it!

**Comment by noyamirai** [comment]
> Styling still missing though. Does anyone have suggestions for the design of the login and register page?

Ahh yes!! Since I have the original design files, I will try to create some design proposals for the needed screens in separate issues later so we can all discuss them together 😄

File: views/register.ejs (hunk 3 above)
Comment (Aidan98) [reply views/register.ejs]: Ahh okay, did not see that, will change.

**Comment by Aidan98** [comment]
> Looks good! So to get the login/create account screens I would simply navigate to `/` right? 🤔 Also, I added some feedback on a few view pages regarding `<head>` tag placements, if you could change that I would appreciate it!

not yet, for now it `/users/login` and `/users/register`. Once I had authentication it will route to `/`

File: views/register.ejs (hunk 3 above)
Comment (noyamirai) [reply views/register.ejs]: pog

**Review by noyamirai — APPROVED** [review]
POG Thank you!!! 🥇

**Pull request merged**